share/locale/musescore_ca@valencia.ts

```diff
@@ -22848,13 +22848,13 @@ compàs d&apos;anacrusi</translation>
         <location filename="../../src/instrumentsscene/view/layoutpaneltreemodel.cpp" line="888"/>
         <source>Are you sure you want to delete the selected %n instrument(s)?</source>
         <extracomment>Please omit `%n` in the translation in this case; it's only there so that you have the possibility to provide translations with the correct numerus form, i.e. to show &quot;instrument&quot; or &quot;instruments&quot; as appropriate.</extracomment>
-        <translation><numerusform>Esteu segur que voleu eliminar l&apos;instrument seleccionat?</numerusform><numerusform>Esteu segur que voleu eliminar els instruments seleccionats?</numerusform></translation>
+        <translation><numerusform>Esteu segur que voleu eliminar l&apos;instrument seleccionat?</numerusform><numerusform>Esteu segur que voleu eliminar els %n instruments seleccionats?</numerusform></translation>
     </message>
     <message numerus="yes">
         <location filename="../../src/instrumentsscene/view/layoutpaneltreemodel.cpp" line="893"/>
         <source>This will remove the %n instrument(s) from the full score and all part scores.</source>
         <extracomment>Please omit `%n` in the translation in this case; it's only there so that you have the possibility to provide translations with the correct numerus form, i.e. to show &quot;instrument&quot; or &quot;instruments&quot; as appropriate.</extracomment>
-        <translation><numerusform>Això eliminarà l&apos;instrument de tota la partitura i de totes les particel·les.</numerusform><numerusform>Això eliminarà els instruments de tota la partitura i de totes les particel·les.</numerusform></translation>
+        <translation><numerusform>Això eliminarà l&apos;instrument de tota la partitura i de totes les particel·les.</numerusform><numerusform>Això eliminarà els %n instruments de tota la partitura i de totes les particel·les.</numerusform></translation>
```

Comment:
The addition of %n here seems not in line with the instructions in `extracomment`... same with the one above.

Reply:
True, removed